lib/fragment.js

```diff
@@ -47,7 +48,7 @@ module.exports = class Fragment extends EventEmitter {
     constructor (tag, context, index, requestFragment, pipeInstanceName) {
         super();
         this.attributes = getAttributes(tag, context);
-        ['async', 'primary', 'public'].forEach((key) => {
+        ['async', 'primary', 'public', 'main'].forEach((key) => {
```

Comment:
the `primary / main` naming is a little confusing. is there some kind of other name we can give this?